File: api/models/exception.js
global.Exception = model.define('Exception', {
    content: {type: Model.STRING, unique: true},
    is_new: {type: Model.BOOLEAN, defaultValue: true},
    treatment: Model.STRING
}, {
    tableName: 'exceptions',
    updatedAt: 'updated_at',
    createdAt: 'created_at'
});

File: website/models/exception.rb
class SendException
  include DataMapper::Resource

  storage_names[:default] = 'exceptions'

  property :id, Serial
  property :content, String, required: true, unique: true
  property :is_new, Boolean
  property :treatment, String

  property :updated_at, DateTime
  property :created_at, DateTime

end